test/OpenSTUtility.js

```diff
@@ -710,13 +710,7 @@ contract('OpenSTUtility', function(accounts) {
 					await OpenSTUtility_utils.checkRevertedMintEvent(result.logs[0], checkBtUuid, stakingIntentHash, accounts[0],
 						accounts[0], AMOUNT_BT);
 				})
-			})
-
-			it('successfully revert Minting', async () => {
```

Comment:
any idea why this test is failing?

Reply:
ah it was duplicated 🤦‍♂️

Reply:
yup
also there were syntax errors with curly brackets while rebasing.